### DISCUSSION TITLE
Onde, quando e como aprender inglês?

### DISCUSSION THREAD

**Discussion by Cleysonlb** [frontendbr/forum#478] [Discussão]
Falae galera, provavelmente já tem uma Issue sobre isso mas queria trazer meu drama...
Passei por todas as escolas de inglês da minha cidade a procura de um lugar que melhor me atende-se para aprender, e enfrentei os seguintes casos, ou era MUITO CARO ou o material era muito fraco e não passou confiança...
Alguém aqui poderia me indicar uma alternativa diferente?
Compartilhar seus próprios casos etc...
Obrigado galerinha vocês são 10! <3

**Comment by luizventurote** [comment]
@Cleysonlb geralmente eu vejo aulas pelo youtube mesmo, vou pesquisando e vou estudando da forma que acho mais produtivo para mim. Dá uma olhada nesses canais, talvez pode te ajudar um pouco:

- https://gist.github.com/luizventurote/2fe0a42a674a0b84e4abf80d61ccf078

Uso o https://www.italki.com/ para fazer aulas com professores nativos de qualquer idioma. Para vocabulário eu uso esse site http://www.fluentu.com/ e também recomendo escutar podcasts de tecnologia ou de qualquer outra coisa que você acha legal.

Bons estudos!

**Comment by fredericksilva** [comment]
@Cleysonlb Acredito que a melhor forma de obter fluencia é convivendo e conversando em inglês, geralmente eu me "forço" a pensar em inglês para tudo, antes de pedir uma informação, ao ler um rotulo de produto no mercado...

E uma coisa que descobri no ano passado foi os meetups de polyglotas ou grupos de conversa do couch surfing foi muito util para acabar com meu medo de iniciar uma conversa por achar que não sou fluente o suficiente.

segue os links:
- https://www.meetup.com/clubepoliglota/events/236761882/ [SP vê se tem um em sua cidade]
- https://www.couchsurfing.com/events

Existe alguns grupos de telegram e whatsapp...

espero ter ajudado em algo :)

**Comment by devigor** [comment]
@Cleysonlb você é dá onde??

**Comment by dionarodrigues** [comment]
Ótimas dicas @luizventurote , vou seguí-las.

Sempre tentei estudar inglês por conta própria, mas nunca consegui avançar muito. Por isso, há alguns anos entrei na wizard e aprendi bastante coisas. 

Ao mesmo tempo comecei a assistir vídeos como esses: https://www.youtube.com/watch?v=NdXPnJLR07E&list=PLhwUb6-mGzbKD2iDp2JtVpcNU6qQCeu3J

Esses vídeos tem me ajudado bastante.

**Comment by kelciocajueiro** [comment]
Pra quem gosta de estudar sozinho:

http://clubedoingles.com/roteiro-de-estudos-para-quem-quer-estudar-ingles-sozinho/
http://clubedoingles.com/rotulo-de-estudo-para-quem-quer-estudar-ingles-sozinho-modulo-2/

**Comment by suissa** [comment]
Quem quer treinar conversação temos um curso focado para Devs!

**[Curso de conversação para programadores](https://github.com/Webschool-io/Curso-de-conversacao-para-programadores)**

**Comment by MendesGabriel** [comment]
Para pronuncia conheci esse site esses dias http://youglish.com/ , você digita a palavra e ele dá vários exemplos de pronuncia no ponto certo do video no youtube.

**Comment by felipefialho** [comment]
#109

**Comment by dsoldera** [comment]
Olá @Cleysonlb. Eu acho que estudar inglês não seja tanto como estudar programação e vc ser 100% didático. Na minha experiência você pode sim utilizar de vários recursos online, como videos youtube e outros para ir sempre melhorando seu vocabulário e gramática, mas precisamos sim de um professor pra avaliar nossa escrita e nossa fala.

Eu ja fiz aula particular, já fiz escolas de inglês  e sempre procurei ver videos e dicas de melhorias. Hoje eu estou estudando pelo English Live (antigo English Town). Não é barato mas são professores com inglês nativo e a plataforma é muito boa (inglês online). Eu estudo em casa e por ja possuir um inglês intermediário tenho me dado muito bem com esse método.

Se o seu inglês for Basico eu não recomendo inglês online. Recomendo sim aulas particulares ou escolas mesmo porque se vc tentar economizar muito pode acabar gastando dinheiro e não caminhar com seu aprendizado. 

É isso.. espero que ajude em algo.... Boa sorte.

**Comment by idaniloreis** [comment]
Fui indicado esses dias a fazer esse, a principio estou gostando bastante e achando interessante, creio que seria uma boa dica:

https://www.tandem.net/

**Comment by markcanals** [comment]
Yo os recomiendo la siguiente página dónde encontrarán recursos gratuitos para aprender inglés de gran calidad y todo enfocado a hispanohablantes:

[lewolang.com](https://www.lewolang.com/gramatica-inglesa)

Espero que les sirva, ya me comentarán que les pareció.

Un saludo,

**Comment by Mauricio-Neviate** [comment]
Eu encontrei o melhor curso gratuito. O professor é super gente boa e envia apostilas gratuitamente. As vídeo aulas são direto ao ponto sem enrolação. É um canal do YouTube chamado Crazy Koller. Vejam o post pelo qual conheci o canal, vou colar aqui:

(*CURSO DE INGLÊS GRATUITO COM MATERIAL INCLUSO*)
Aula 01: https://www.youtube.com/watch?v=KTDWjLxBcM8&t=3s

*Curso de inglês gratuito. Siga as aulas na sequência.
*Se inscreva no canal e receba material em PDF baseado nas vídeo aulas.
*Agora também com MP3.
*Siga as instruções na descrição dos vídeos para baixar o material gratuitamente.

*Os vídeos são em inglês. POR FAVOR ATIVE A LEGENDA
Método:

1- assista os vídeos com legenda em português.
2- assista novamente com legenda em inglês.
3- assista novamente sem legenda.
4- baixe o PDF da aula assistida e faça os exercícios.

*Por favor, ajudem a continuar com esse projeto compartilhando com seus contatos e em grupos e pedindo que eles compartilhem* (a intenção é manter a gratuidade deste curso)

(Toda semana uma aula nova, com material novo a cada aula nova)

**Comment by carloshenriqueribeiro** [comment]
Muito bom man! Ja comecei a seguir é curtir os vídeos.

**Comment by carloshenriqueribeiro** [comment]
@suissa ainda existe ? Lá ta com início dez 2016. Abraço

**Comment by tgmarinho** [comment]
Olá, eu faço curso com www.jacksonrogeridiomas.com.br  (pago) é bem interessante o método, mas é cansativo também, uso Duolingo e algumas outras ferramentas estão aqui:

https://github.com/tgmarinho/english-learning-tools

**Comment by woliveiras** [comment]
Galera, a um tempo atrás eu fiz um artigo sobre os apps que estou usando pra aprender/praticar inglês (de gratis): [Aplicativos para treinar nosso inglês](https://medium.com/trainingcenter/aplicativos-para-treinar-nosso-ingl%C3%AAs-c957d2c6dc9f)

**Comment by Murillofilho86** [comment]
Depois de procurar alguns cursos, descobri que a melhor opção é contratar um professor particular... Faço minhas aulas por skype, na hora que for melhor pra mim, não tive que gastar rios de dinheiro com material, pago mais barato que um curso... Estou indo pro meu terceiro mês e já percebo uma melhora muito significativa... Minha leitura e pronuncia melhoraram bastante, mas ainda tenho dificuldades em formar textos complexos e mesmo assim, estou melhor do que na época em que comecei.
Fica a minha dica, procure um professor particular, mas lembre-se de checar as referencias do mesmo.

**Comment by Marlysson** [comment]
@Murillofilho86 por onde encontrou esses professores particulares? São nativos mesmo ou não-nativos mas com grande experiência no idioma?

E com relação ao preço... tem valores..?

**Comment by mahenrique94** [comment]
Recomendo a [English Live](https://englishlive.ef.com/pt-br/), [Alura Lingua](https://www.aluralingua.com.br/) e [Duolingo](https://pt.duolingo.com/).

Além claro de muita prática, particularmente, tento sempre utilizar o inglês, meu note, celular, programas, sites, projetos, códigos, tudo, praticamente tudo que der eu tento fazer com inglês, assim consigo praticar bastante.

Também gosto de assistir vídeos, séries ou filmes com audio e legenda em inglês (futuramente pretendo deixar apenas audio e sem legenda).

O segredo é praticar bastante, é melhor praticar um pouco todos os dais, do que bastante mas apenas um dia ou outro.

**Comment by Murillofilho86** [comment]
@Marlysson, foi indicação de uma amiga, ela também da aulas de inglês, porém, presencial... E como eu queria online, ela me indicou a prima, que é professora formada, trabalha com isso e tal... 
Ela trabalha com pacotes, os valores vão variar de acordo com a quantidade de horas por semana.

ps. desculpa a demora pra responder leke, mas acho que deixei passar a notificação dessa conversa.

**Comment by eliseumds** [comment]
Se não houver meetups na sua cidade, crie um! Eu falo fluentemente há um tempo mas ainda vou a meetups direto aqui em Sydney pra poder ajudar a galera. Vi o mesmo acontecendo em Sampa. É 200x melhor que qualquer curso, se você encontrar alguém fluente.

**Comment by didiraja** [comment]
@eliseumds meetup de conversacao? nunca tinha ouvido falar, boa ideia.

**Comment by danilocandido** [comment]
Em Fortaleza tem todo sábado a partir das 17:00. Sem falta.
https://www.meetup.com/pt-BR/Clube-Poliglota-Fortaleza/

**Comment by BrunoMarc2** [comment]
Olá! Dentre os aplicativo gratis eu recomendo o duoling, é muito bom eu já usei. Conheço um outro método também muito bom, (bem ousado) que usa o método Accelerated Learning ( desenvolvida pela Universidade de Harvard ) segue o link: **[Como Aprender Inglês Rápido e Sozinho](https://oradorninja.com/como-aprender-ingles-rapido-sozinho-funciona/)** , espero que seja útil! Abs

**Comment by Vasmi238** [comment]
Recomendo o https://languagetools.io/. É um hub bem completo, vc pode ler textos com tradutor e pronúncia automáticos, escrever e ter seus essays corrigidos por professores, participar de chats, enviar threads com dúvidas, participar de discussões, receber dicas e, o melhor, trocar com pessoas de diversas nacionalidades, pois não é exclusivo para quem está aprendendo Inglês, apesar de ser a língua principal por ali. Uso bastante e está sendo muito bom pro meu desenvolvimento!

**Comment by Mauricio-Neviate** [comment]
O canal do Youtube RODGER KOLLER tem um curso sequencial e ele envia o
material das vídeo aulas gratuitamente para quem solicita.

Em sex., 27 de dez. de 2019 às 02:06, Vasmi238 <notifications@github.com>
escreveu:

**Comment by jair-anderson-souza** [comment]
Tô pensando em fazer o do Mairo Vergara, alguém já fez? Será se vale a pena?

**Comment by CristinaMarin05** [comment]
Gente vcs tem conta na Amazon? Ela disponibilizou um livro super barato e prático auto didático para ficar fluente em 6 meses, comprei o meu nesse link https://www.amazon.com.br/gp/product/B01HJAL6Z6/ref=as_li_tl?ie=UTF8&camp=1789&creative=9325&creativeASIN=B01HJAL6Z6&linkCode=as2&tag=travel0770f-20&linkId=06dc3cc858f7f9eddd5b2313b24e68f2
Espero que ajude quem quiser tb!!

**Comment by dionarodrigues** [comment]
Hello everyone!
Após alguns anos depois que eu postei como eu vinha aprendendo inglês, volto aqui pra dizer como consegui dar um salto no meu aprendizado.

Em 2018 decidi fazer um intercâmbio para o Canadá. Por isso, com o intuito de aprender o máximo possível antes de ir estudar na gringa, comecei a devorar conteúdos sobre gramática e isso me ajudou muito nos 6 meses anteriores à minha viagem. Na escola que estudei lá no Canadá (ILAC - indico muito) há 17 níveis, do básico ao avançado. Quando fiz um teste online eu estava no nível 4. Quando cheguei lá, após os meses de preparação antes de viajar, eu já estava no nível 10. 

O segredo foi assistir bastante vídeos explicando regras gramaticais básicas, porém tudo em inglês (com professores explicando apenas em inglês). 

Os  vídeos desse canal me ajudaram muito (nível bem iniciante): https://www.youtube.com/watch?v=AVYfyTvc9KY&list=PL2IkMHFHWdErN1-V9nAeaxXiSkG7kBtjS

Esse indiano é phoda e aprendi muito com os vídeos dele (um pouco mais avançado): https://www.youtube.com/channel/UCImiNBzWUNuHBm95F8SyMYA

Em seguida vem o pessoal da EngVid que também me ajudaram muito a consolidar meus conhecimentos:
- Ronnie: https://www.youtube.com/user/EnglishLessons4U
- James: https://www.youtube.com/user/JamesESL
- Emma: https://www.youtube.com/user/EnglishTeacherEmma
- Adam: https://www.youtube.com/user/EnglishTeacherAdam
O legal da EngVid é que tem muito mais professores e vocês podem fazer pequenos testes após cada aula pra treinar a fixação (e tudo gratuito e sem login): https://www.engvid.com/

Essa Rachel me ajudou muito a melhorar a pronúncia (tem coisas avançadas, mas também tem muito vídeos básicos): https://www.youtube.com/user/rachelsenglish

Também tive algumas horas de conversação com pessoas diversas do mundo todo através da plataforma iTalk: https://www.italki.com/ (é possível filtrar por preço/hora, escolher pessoas que não sejam brasileiras pra evitar falar em português, assistir o vídeo demo pra ver se o sotaque do "professor" vai ser fácil de entender...)

Bom, e havia muito mais. Muitos aplicativos como Duolingo por exemplo e outros. Fiz do estudo do inglês um hábito diário: estudava todos os dias (anotando as dicas e reescrevendo o que eu lia) e ficava revisando o dia todo. Ficava assistindo vídeos em inglês (de acordo com o menu nível), lia posts em inglês, ficava repetindo as palavras que eu ouvia... E tentei focar apenas no inglês americano, uma vez que eu ia para o Canadá.

Bom, e após 4 meses no Canadá, voltei com o meu objetivo concluído: conseguir consumir conteúdos da área de desenvolvimento em inglês - vídeos, posts, podcasts... Claro que lá eu aprendi muito. A escola era ótima. Me enturmei com pessoas de diversas nacionalidades o que me fez falar muito em inglês. Tive muita sorte com as duas famílias que morei, pois pude aprender muito com eles... Claro que eu também fiz amigos brasileiros e continuamos a trocar mensagens até hoje, inclusive em inglês...

É uma experiência única e se você conseguir ir estudar pelo menos 2 semanas em algum outro país e focar ao máximo em aprender e praticar enquanto estiver lá, com certeza seu inglês vai decolar ainda mais.

Espero ter ajudado!
Super abraço.

**Comment by abechuate** [comment]
Olá Pessoal, gostaria de convidar-los para um zoom call para falar sobre o assunto aprender inglês. Acredito que vai ser bastante valioso! 
Não sou professor de inglês, não tenho ou represento nenhuma escola, mas tenho uma uma longa e intensa experiência pessoal com o Inglês. Ficaria feliz em poder ajudar.
Quem tiver interesse, por favor, envie um email para englishmastery@liveup2yourdreams.com e agendamos uma data.
Um abraço,
André

**Comment by Mauricio-Neviate** [comment]
Eu gosto do canal Rodger Koller

Em ter., 5 de mai. de 2020 às 20:39, abechuate <notifications@github.com>
escreveu:

**Comment by abechuate** [comment]
Olá Mauricio,

Vou fazer outro webinar muito em breve para apresentar meu programa de
mentoring. Se tiver interesse, deixe-me saber!

Um abraço,
André

**Comment by agamenor** [comment]
olha esse site de [aulas particulares](https://lumaescola.com.br)

**Comment by lucasrmagalhaes** [comment]
Eu uso o app Duolingo ou o app Drops.

**Comment by youenglishofficial** [comment]
Hey there mate!

Aprender inglês é duplamente mais difícil quando se tem que buscar preços acessíveis combinados com uma qualidade aceitável. Além de tudo, é super importante também levar em consideração a conexão que você estabelece com seu professor, afinal é ele quem vai te levar por essa brilhante jornada.

Acima de tudo, vale considerar o inglês como um investimento, e como todo o investimento, considerar o (momento aula de inglês rsrs) R.O.I., ou Return Of Investment. Muitas vezes o que parece ser caro hoje vai te gerar frutos muito maiores num futuro a médio e longo prazo! Eu sei que combinar isso com as contas do mês não é fácil, mas muitas vezes falhamos em considerar isso.

Possuo um canal de inglês no YouTube, www.youtube.com/c/YouEnglish, assim como no Instagram, @_youenglish_, com vídeos semanais de aulas gramaticais e linguagem, espero que te ajude! E claro, tem que curtir, se inscrever e deixar comentários praticando o que foi ensinado em aula, ok? Prometo que vou ler todos e opinar!

Um grande abraço!

**Reply by fabiochiquezi** [reply]
Não sei como vim parar num tópico tão antigo haha, mas +1 inscrito

**Comment by Robom77** [comment]
Então, eu estudo sozinha por youtube, blogs de ESL que acompanho e também pratico com cambly e outros meios.

Tem um blog que tem muito conteúdo e exercícios que costumo frequentar: https://fluencygo.com/

**Comment by brunobraga95** [comment]
Edit: repostando como reply da thread e não do último comentário 

Olá pessoal

Moro na alemanha e recentemente comecei a ouvir podcast para me ajudar com a língua. O podcast exato se chama easygerman e eu gostei muito do modo com que eles disponibilizam transcripts para que o áudio possa ser acompanhado com leitura.

Com isso em mente e tempo livre no fim de semana, eu decidi criar uma plataforma parecida, mas focada no público brasileiro que está interessado em inglês. Eu gostaria de saber o que vocês acham e caso conseguem ver essas notícias ajudando algumas pessoas.

O principal benefício é a possibilidade de tanto se manter atualizado, quanto aprender ou reforçar o inglês. No meu protótipo inicial eu adicionei features como acompanhamento de leitura e tradução simultânea quando uma palavra ou expressão é clicada. No futuro pretendo adicionar quizzes, lista de palavras favoritas e Flash cards.

Hoje os principal benefícios são:

Poder acompanhar o áudio junto com o texto, isso auxilia muito no foco.Ter acesso fácil as traduções, não precisar lidar com duas abas ou múltiplos cliques.Aprender pronúncia, o que não acontece em uma leitura simples.

Eu acabei de terminar a minha primeira notícia e gostaria de saber o que vocês acham, dependendo da resposta vou decidir se vale a pela ou não prosseguir.

Qualquer feedback é extremamente bem-vindo, meu plano é que o site seja gratuito e meu maior objetivo é ver pessoas usando e conseguir ajudar alguns seres humanos :)

https://www.news-lang.com/news/renan-decided-to-delay-to-october-covid-cpi-final-report

**Comment by grtdoingles** [comment]
Sei de um curso muito bem avaliado do básico ao avançado com uma professora nativa e com acesso vitalício, dizem que é muito bom.

Dá uma olhada, é bem mais barato que a maioria dos cursos por aí, e ainda tem a vantagem da professora ser nativa dos EUA.

https://hotm.art/ingles-com-a-gringa-conheca

E pra te ajudar, tem esse e-book grátis bem legal também  https://bit.ly/11-dicas-ingles-gringa

**Comment by felipe-heredia** [comment]
Recentemente eu descobri o jogo [NationStates](https://nationstates.net), segundo o site do jogo:

> _NationStates is a free nation simulation game. You create your own country, fashioned after your own ideals, and care for its people. Either that or you deliberately torture them. It's really up to you._

De tempos em tempos o jogo te entrega _issues_, que são problemas que você precisa resolver dentro da sua nação. 

Com cada problema são entregues algumas opiniões de pessoas (já vi um máximo de 6) e você vai escolher com qual concordar e assim ser o posicionamento oficial da nação sobre o assunto.

Aqui vai uma definição do site que explica o que são essas Issue:

> _A few times a day, you'll face [an issue](https://www.nationstates.net/page=dilemmas) and need to decide what to do about it. How you respond determines almost everything about how your nation evolves. There's no micro-management. You control your nation's destiny by making broad policy decisions, not tweaking numbers._

Os problemas são realmente extensos, vão de 'Chocolate que estimulam as mulheres devem ser proibidos?' até 'Os presos estão fugindo da prisão após uma chuva muito forte'.

Pelo fato de te entregar problemas realmente distintos, você vai conseguir resolver problemas (em inglês) no qual você jamais imaginaria, assim você pode aprender palavras novas que jamais aprenderia em um curso.

PS.: Eu recomendo ler todas as opções para a resolução, mesmo que você escolha a primeira, se existirem 6 resoluções, leia as 6 e aprenda novas palavras.

---

Por fora do jogo, um dos meus objetivos é conseguir aprender 10 palavras todos os dias, o jogo ajuda nisso, mas conteúdos técnicos também.

Gosto de anotar as palavras no Notion, pois consigo ter um controle melhor das palavras que aprendi, segue um exemplo da organização:

![2022-02-10_16-21](https://user-images.githubusercontent.com/50281523/153481274-21e1ee4c-2c53-4e1e-8ba8-3146f0770e5c.png)

Com essa organização, consigo contar quantas palavras já aprendi, os dias etc. Consigo também buscar no 'banco de dados' se a palavra que eu quero escrever já consta assim eu não coloco ela como um novo item.

PS.: Utilizo um Dicionário pra saber o que a palavra significa e não um tradutor.

**Comment by Sias09** [comment]
Pessoal, 
Hoje sou fluente porém já passei muito perrengue em diversas escolas que não aprendi nada... só tomou meu dinheiro e meu tempo. Além do foco ser gramática os professores falavam muito em português... Saia da sala me sentindo decepcionado e frustado! Até que um amigo meu me indicou uma escola onde o foco era aprender a falar inglês da mesma forma que aprendi o português. Comecei do zero novamente na HC School (www.hcschool.online)  porque minha pronúncia era um lixo. Sofri com isso pq estudei por três anos e descobri que não falava nada, até conhecer essa forma de aprendizado. 
A HC foca muito no desenvolvimento da fala e correção da pronúncia. Meus professores eram top e super recomendo. 
A escola oferece uma aula experimental grátis, sugiro que faça para conhecer.

**Reply by vczb** [reply]
> Pessoal, Hoje sou fluente porém já passei muito perrengue em diversas escolas que não aprendi nada... só tomou meu dinheiro e meu tempo. Além do foco ser gramática os professores falavam muito em português... Saia da sala me sentindo decepcionado e frustado! Até que um amigo meu me indicou uma escola onde o foco era aprender a falar inglês da mesma forma que aprendi o português. Comecei do zero novamente na HC School ([www.hcschool.online](http://www.hcschool.online)) porque minha pronúncia era um lixo. Sofri com isso pq estudei por três anos e descobri que não falava nada, até conhecer essa forma de aprendizado. A HC foca muito no desenvolvimento da fala e correção da pronúncia. Meus professores eram top e super recomendo. A escola oferece uma aula experimental grátis, sugiro que faça para conhecer.

Favoritei aqui, depois vou dar uma olhada! vlws =)

**Comment by OnMindapp** [comment]
Treine seu inglês dormindo
Estude inglês grátis
Aproveite melhor o seu tempo e aprenda um novo idioma de forma definitiva
Com o OnMind você pode aprender inglês até mesmo dormindo
No OnMind você irá aprimorar seu vocabulário de forma prática, rápida e totalmente grátis
Memorizar palavras de um novo idioma nunca foi tão fácil!
Memorize palavras ou frases inteiras utilizando o Sistema de Repetição Espaçada do OnMind!

https://onmind-idiom.web.app/

**Comment by marcelomedeirosrizza** [comment]
> Falae galera, provavelmente já tem uma Issue sobre isso mas queria trazer meu drama... Passei por todas as escolas de inglês da minha cidade a procura de um lugar que melhor me atende-se para aprender, e enfrentei os seguintes casos, ou era MUITO CARO ou o material era muito fraco e não passou confiança... Alguém aqui poderia me indicar uma alternativa diferente? Compartilhar seus próprios casos etc... Obrigado galerinha vocês são 10! <3

Pessoal, para quem precisa apreender inglês de forma rápida e efetiva, eu recomendo MUITO este curso de inglês online aqui https://www.youtube.com/watch?v=Dcli_p4Y-7w GRAVEI um vídeo explicando como tudo funciona POR DENTRO.

O mais LEGAL é que você pode fazer aulas AO VIVO 24 horas por dia 7 dias por semana UMA ATRÁS DA OUTRA se desejar.

Esta é a forma mais PRATICA e INTELIGENTE de apreender inglês para conseguir aquela vaga de trabalho ou fazer um verdadeiro
INTERCAMBIO sem sair de casa.

Eu faço a 2 anos e fico a disposição para ajudar respondendo a dúvidas!

**Reply by marcelomedeirosrizza** [reply]
Pesquisei MUITO! ai fiz minha decisão e ACERTEI em cheio. Preço JUSTO com aulas ao vivo 24 horas por dia. Aulas particulares e em grupo. Vale cada centavo. Da para fazer uma verdadeira IMERSÃO sem sair de casa.

**Comment by BrazilianEagle** [comment]
Pessoal, aprender ou aperfeiçoar seu inglês não precisa ser difícil e desesperador! 

Aqui estão algumas dicas valiosas, que fui adquirindo e me ajudaram muito ao longo do tempo:
	1.	Cinema em Inglês, a Sessão que Você Nunca Esquecerá:
	◦	Mergulhe em filmes e séries em inglês. Não apenas você se entretém, mas absorve a autenticidade do idioma e aprende expressões cotidianas de forma natural.
	2.	Viaje Pelas Páginas de Livros em Inglês:
	◦	Deixe-se envolver por histórias cativantes em inglês. A leitura amplia seu vocabulário, melhora a gramática e proporciona uma compreensão mais profunda da cultura anglo-saxã.
	3.	Podcasts - O Seu Companheiro de Aprendizado:
	◦	Ouça podcasts em inglês durante suas atividades diárias. É uma maneira eficiente de treinar seu ouvido para diferentes sotaques e contextos linguísticos.
	4.	A Imensidão do YouTube para o Seu Progresso:
	◦	Explore canais educativos em inglês no YouTube. Há uma riqueza de conteúdo para todos os níveis de habilidade, desde gramática até dicas para pronúncia.
	
Estas são apenas algumas dicas para atingir a maestria do inglês, mas certamente uma orientação personalizada e dedicada encurtará bastante esse caminho! 

Nesse caso, recomendo o professor que fiz aula por um bom tempo Giovanni. Ele oferece um ambiente de aprendizado acolhedor e eficaz, possibilitando que você adquira fluência em 8 meses de aula (aqui sem rodeios, para sair fluente você tem que querer, e aconselho a fazer aula no mínimo 2x na semana com ele para ter uma boa base). 

Entrando em contato, pode falar que foi recomendado pelo "Brazilian Eagle"! Ele vai saber quem é. haha   

Abraço.

**Comment by Wiltp** [comment]
essoal, vocês já ouviram falar no curso "A Jornada do Autodidata em Inglês" do Gabriel Becker? Acabei de conhecer e fiquei impressionado com a proposta! 👏
[https://officialnow1.com/a-jornada-do-autodidata-em-ingles/](url)

O método deles é baseado no Inglês 80/20, ou seja, o foco está nos 20% do idioma que realmente são usados no dia a dia pelos nativos. Faz muito sentido, porque nós também não usamos 100% do português, né? E o mais legal é que dá pra alcançar fluência em 3 meses com dedicação, sem precisar gastar horrores em cursinhos tradicionais.

Alguns diferenciais que chamaram minha atenção:

Aulas de conversação ao vivo toda semana com o próprio Gabriel. Isso é raro, porque a maioria dos cursos online só tem vídeo gravado.
Acesso vitalício, então dá pra rever o conteúdo quantas vezes quiser sem pressa.
Reconhecimento pelo MEC, o que dá um peso extra pro certificado.
Material de apoio gratuito e biblioteca de áudios com transcrições!
Além disso, o curso promete ser super dinâmico, com aquele "Efeito Netflix" que te deixa viciado em estudar. Isso resolve o problema de desmotivação que muita gente enfrenta com cursos tradicionais.

Vi depoimentos de alunos que mudaram a vida com o curso, como um senhor de 63 anos que agora negocia em inglês e uma jovem que aprendeu a assistir séries sem legenda em poucos meses. Parece muito prático e eficiente.

Estou considerando me inscrever. Alguém aqui já fez ou conhece alguém que fez? Quero saber mais opiniões! 😊

**Reply by matheus-arruda** [reply]
Não caia nessa meu amigo, se o cara me promete uma fluência em 3 meses eu já pulo fora! Na minha cabeça isso é impossível!

Fiz o curso do Teacher Matias, é um curso bem legal e tem aulas de conversação além de encontros com o proprio Matias, porém precisei de um professor para ter mais vivência com o idioma, comecei o curso dele em Agosto/2023 e hoje eu já faço aulas com um professor que é americano nativo e não fala nada de Português. Nesses 1,3 anos sai do zero a um B2 sólido e todos os dias eu estudo pelo menos uma hora, agora tá dificil sair do intermediario estou pensando em contratar alguma mentoria para alavancar isso.

Se alguém precisar tenho alguns recursos para conversar com outras pessoas sem custo:

**Conversation in the group:** 
Langclub: https://langclub.live/sessions
Promova: https://promova.com/conversation-club

**One one conversation:**
free4talk: https://www.free4talk.com/
Episoden: https://www.episoden.com/

Last one, chat gpt: https://chatgpt.com/

Uma outra coisa que gosto de fazer é gravar videos as vezes e postar no youtube só pra perder a vergonha mesmo.
